README.md

```diff
@@ -58,7 +58,7 @@ depending hardfork parameters. There are also additional helper functions like
 `paramByBlock (topic, name, blockNumber)` or `hardforkIsActiveOnBlock (hardfork, blockNumber)`
 to ease `blockNumber` based access to parameters.
 
-- [API Docs](./docs/Readme.md)
+- [API Docs](./blob/master/docs/README.md)
```

Comment:
Curious as to the reason behind this change, as the new path is probably github-specific.

Reply:
The link was not working and I couldn't get it to work, tried https://github.com/ethereumjs/ethereumjs-common/docs/README.md. Am I missing something? Definitely not optimal that this is GitHub specific then.

Reply:
The absolute link doesn't work you're right, maybe when its in the markdown document github will detect it? something like `[API Docs](./docs/README.md)` instead of `Readme.md`. Checked `ethereumjs-util`, the link there is also `./docs/index.md`

Reply:
Ok, then I'll use `./docs/README.md` (so a change to capital letters on the file name) and hope GitHub knows what to do.

Reply:
Have updated.

Reply:
The updated path `./docs/README.md` works correctly.